src/common/containers/GameArea.js

```diff
@@ -577,14 +597,20 @@ export class GameArea extends Component {
             onClick={this.handleClickEndGame} />
         </Paper>
 
-        <Chat
-          inGame
-          fullscreen={screenfull.isFullscreen}
-          open={this.state.chatOpen}
-          toggleChat={this.toggleChat}
-          roomName={this.props.socket.hosting ? null : this.props.socket.gameName}
-          messages={this.props.socket.chatMessages.concat(this.props.actionLog)}
-          onSendMessage={this.props.onSendChatMessage} />
+        {
+          this.props.isSandbox ?
```

Comment:
I think these lines are too long for a ternary. Maybe a `renderCardSelector` and `renderChat`?

Reply:
I'll fix this after I merge in the GameArea refactor PR – I don't want to make the diff too messy right now.